doc/conf.py

```diff
@@ -92,11 +92,11 @@ def _check_deps():
 autodoc_default_flags = ['members', 'undoc-members']
 
 intersphinx_mapping = {
-  'python': ('https://docs.python.org/', None),
-  'numpy': ('https://docs.scipy.org/doc/numpy/', None),
-  'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None),
-  'pandas': ('http://pandas.pydata.org/pandas-docs/stable', None)
-  }
+    'python': ('https://docs.python.org/3', None),
+    'numpy': ('https://docs.scipy.org/doc/numpy/', None),
+    'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None),
+    'pandas': ('http://pandas.pydata.org/pandas-docs/stable', None)
```

Comment:
Seems like they support secure connections now too, if you want to try that out.